ports/stm/boards/stm32f411ce_Iskra_JS_mini/mpconfigboard.mk

```diff
@@ -0,0 +1,22 @@
+USB_VID = 0x239A
```

Comment:
This is the Adafruit vendor ID and shouldn't be used for non-Adafruit boards without permission.

Please ask the manufacturer to provide a PID under their VID or request one from http://pid.codes.

Reply:
The manufacturer is not responding, can I use USB_VID = 0x0483 USB_PID = 0x5740 STMicroelectronics Virtual COM Port?

Reply:
I'd rather not. I'd prefer a pid.codes ID.